selene-lib/src/ast_util/mod.rs

```diff
@@ -37,6 +41,34 @@ pub fn first_code<'ast>(ast: &Ast<'ast>) -> Option<(Position, Position)> {
     }
 }
 
+pub fn is_vararg(expression: &ast::Expression) -> bool {
+    if_chain::if_chain! {
+        if let ast::Expression::Value { value, .. } = expression;
+        if let ast::Value::Symbol(token) = &**value;
+        if let tokenizer::TokenType::Symbol { symbol } = token.token().token_type();
+        if let tokenizer::Symbol::Ellipse = symbol;
+
+        then {
+            true
+        } else {
+            false
+        }
+    }
+}
+
+pub fn is_function_call(expression: &ast::Expression) -> bool {
+    if_chain::if_chain! {
+        if let ast::Expression::Value { value, .. } = expression;
+        if let ast::Value::FunctionCall(_) = &**value;

```

Comment:
This isn't enough for an if_chain, the other one was 4 long so it made more sense.